packages/client/webpack.config.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Disabling eslint won't be necessary when we use `import`, eg.: `import path = require('path');`. If we don't want to use imports we can just remove the rule from the config.

Reply:
This isn’t for typescript this is for the webpack.config.js file.  I ignored the imports because we are not running as type=module.